### DISCUSSION TITLE
Windows version

### DISCUSSION THREAD

**Discussion by mvdgeijn** [beyondcode/herd-community#331] [Q&A] [answered]
Hi,

Where can I find the Herd Pro windows version? Taylor Otwel said 2 days ago during the Laracon EU that Herd Pro would be available. Or was there some misunderstanding?

**Comment by mechelon** [comment, marked as answer]
Herd Pro - for MacOS - is available now. The Windows version (Herd Basic + Pro) will be available in a few weeks. If you're interested in that, you can go to herd.laravel.com on a Windows machine and get signed up for the launch list.
